learning-another-language/fizzbuzz.md

```diff
@@ -0,0 +1,258 @@
+# FizzBuzz
+
+## Introduction
+
+When learning another language, implementing FizzBuzz is a great way to jump into it. This program allows us to explore how to:
+
+- use variables
+- use conditionals
+- use equality-checking and comparison
+- define functions
+- use parameters
+- use `return`
+- call functions
+
+The requirements of the FizzBuzz program is as follows:
+
+Create a function named `fizzBuzz`. It takes in one parameter, `num`.
+
+- If `num` is a multiple of 3, then return `'Fizz'`.
+- If `num` is a multiple of 5, then return `'Buzz'`.
+- If `num` is a multiple of 3 _and_ 5, then return `FizzBuzz`.
+- If `num` is not a multiple of 3 or 5, then return the number itself.
+
+## Implementation
+
+[To implement FizzBuzz, we can write](https://replit.com/@adacore/FizzBuzz):
+
+<!-- prettier-ignore-start -->
+```js
+const fizzBuzz = function(num) {
+  if (num % 15 == 0) {
+    return 'FizzBuzz';
+  } else if (num % 3 == 0) {
+    return 'Fizz';
+  } else if (num % 5 == 0) {
+    return 'Buzz';
+  } else {
+    return num;
+  }
+}
+
+console.log(fizzBuzz(1));
+console.log(fizzBuzz(3));
+console.log(fizzBuzz(5));
+console.log(fizzBuzz(15));
+```
+<!-- prettier-ignore-end -->
+
+[When we run our code](https://replit.com/@adacore/FizzBuzz), we should see this output:
+
+```
+1
+Fizz
+Buzz
+FizzBuzz
+```
+
+### Variable Assignment and Function Definition
+
+Let's dive into this fragment:
+
+```js
+const fizzBuzz = function(num) {
+```
+
+| <div style="min-width:200px;"> Piece of Code </div> | Notes                                                                                                                                   |
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `const fizzBuzz`                                    | This _declares_ a variable named `fizzBuzz`. This specific type of variable is a `const` variable. The variable's name is in camelCase. |
+| `=`                                                 | This is the assignment operator. This will assign whatever value is on the right-side to the variable on the left side, `fizzBuzz`.     |
+| `function`                                          | The keyword to begin the definition of a function                                                                                       |
```

Comment:
here, this is a function expression. I think we can add that small amount of precision without it being confusing
```suggestion
| `function`                                          | The keyword to begin the definition of a function expression                                                                                      |
```